_episodes/02-markdown.md

```diff
@@ -221,8 +221,46 @@ keypoints:
 {: .challenge }
 
 
+Linking other websites is an essential feature of a website.
+Bare URLs like https://carpentries.org/ can be made clickable by wrapping them into 
+angle brackets like this <https://carpentries.org/>.
+However, usually the bare URL is not very nice and displaying a clickable
+alternative text is beneficial.  
+You can define either
+1. Inline style links - specifying the URL in line - or
+2. Reference style links - referencing a reusable link reference -
 
 
+> ## Links
+> Fill in the blanks below to create this rendered view
+> 
+> Examples:
+> 1. [Inline link](https://carpentries.org/)
+> 2. [Reference link][case-InSeNsiTiVe-reference-tag]
+>
+> [case-insensitive-reference-tag]: https://carpentries.org/
+>
+> ~~~
+> Examples:
+> 1. [___](https://carpentries.org/)
+> 2. [___]___case-InSeNsiTiVe-reference-tag___
+>
+> [case-insensitive-reference-tag]: https://carpentries.org/
+> ~~~
+> {: .source }
+>
+> > ## Solution
+> > ~~~
+> > Examples:
+> > 1. [Inline link](https://carpentries.org/)
+> > 2. [Reference link][case-InSeNsiTiVe-reference-tag]
+> >
+> > [case-insensitive-reference-tag]: https://carpentries.org/
```

Comment:
Do these get rendered as links in the lesson or does the code black keep them as plain text?  One of the complications of teaching markdown in a markdown lesson. ;)

Reply:
Ignore this comment.  the first exercise in this lesson seems to be fine so it looks like it won't render the md inside a code block.